teacher.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person'
+
+class Teacher < Person
+  attr_accessor :specialization
+
+  def initialize(id, age, specialization, name: 'Unknown', parent_permission: false)
+    super(id, age, name: name, parent_permission: parent_permission)
```

Comment:
- [ ] [Optional] [Highly recommended] If you implement the change I recommended above for the `id`, Please remove the id here from lines 6 and 7.